nc_time_axis/__init__.py

```diff
@@ -248,17 +248,21 @@ def convert(cls, value, unit, axis):
                 return value
             first_value = value
 
+        if not isinstance(first_value, CalendarDateTime):
+            msg = 'The value must be a number, a sequence of numbers or of ' \
+                  'type :class:`nc_time_axis.CalendarDateTime.'
```

Comment:
@lbdreyer Dah! You're missing the closing tick around `nc_time_axis.CalendarDateTime`.

You could avoid the whole issue and instead say:
`... sequence of numbers or an instance of "nc_time_axis.CalendarDateTime"`

The usage of `:class:...` is Sphinx-like markup that won't be rendered differently, but shown raw as-is when this exception is raised.

Change either way, your choice :wink: